### PR TITLE
fix: move porfolio button next to price amount

### PR DESCRIPTION
**Pull request by jonybur**
## **Description**

Moves the portfolio button so that it's next to the token price amount.

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/26867?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/3201

## **Manual testing steps**

1. Go to the portfolio page
2. Check that the Portfolio button is present next to the price amount of token
3. Click it to test it works fine

## **Screenshots/Recordings**

Screenshot 2024-09-03 at 22 57 32
Screenshot 2024-09-03 at 22 58 54

### **Before**

Screenshot 2024-09-03 at 15 46 36

### **After**

Screenshot 2024-09-03 at 22 58 54

## **Pre-merge author checklist**

- [X] I've followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Extension Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [X] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [X] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [X] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [X] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: ui/components/app/wallet-overview/index.scss [hunk 1]
```diff
@@ -160,4 +166,4 @@
   &__button:last-of-type {
     margin-right: 0;
   }
-}
+}
```
Comment (EtherWizard33) [inline ui/components/app/wallet-overview/index.scss:169]: nit: maybe that end of file isn't expected?

**Review by EtherWizard33 — DISMISSED** [review]
LGTM

**Review by NidhiKJha — DISMISSED** [review]

**Comment by hesterbruikman** [comment]
@jonybur so sorry, we have a new change request while you still have this PR open; We'd also want to change the order of the buttons. 

Buy/Sell
Swap
Bridge
Send
Receive

cc @mmragkandala

**Comment by mmragkandala** [comment]
I've also updated the [designs in figma](https://www.figma.com/design/DXoMqztDs01OGOSaMRgwJF/Action-menu-%26-Wallet-header?m=auto&node-id=3386-1814&t=wCqLoeXydwV6urAG-1) with this new ordering

image

**Review by EtherWizard33 — APPROVED** [review]
LGTM

**Review by NidhiKJha — APPROVED** [review]

**Pull request merged**